src/Pelagos/Bundle/AppBundle/Resources/views/template/AdminOnly.html.twig

```diff
@@ -1,4 +1,5 @@
-<div class="messages error">
+<div class="messages error alert alert-danger">

```

Comment:
Again, not related to this pull request. 
Non-DRPM users are able to access DRPM pages